sycl/doc/design/SYCLNativeCPU.md

```diff
@@ -50,9 +69,19 @@ cmake \
 
 Note that a number of `e2e` tests are currently still failing.
 
-# Running example
+## Ongoing work
 
-The following section gives a brief overview of how a simple SYCL application is compiled for the Native CPU target. Consider the following SYCL sample, which performs vector addition using USM:
+* Complete support for remaining SYCL features, including but not limited to
```

Comment:
At this stage we'd like to keep the option of at least being able to compile trivial applications without the OCK, but yeah it the medium term it would be good to drop the mode without the OCK since it would simplify testing and maintenance, @uwedolinsky what do you think?

Reply:
I agree. We only test with OCK enabled going forward so it makes sense dropping the mode without OCK.